lib/Webservice/Shipment/Carrier.pm

```diff
@@ -7,13 +7,13 @@ use Mojo::UserAgent;
 use Mojo::IOLoop;
 
 use Carp;
-our @CARP_NOT = ('Webservice::Shipment'); # don't carp from AUTOLOAD
 
 has api_url => sub { Mojo::URL->new };
 has password => sub { croak 'password is required' };
 has ua => sub { Mojo::UserAgent->new };
 has username => sub { croak 'username is required' };
 has [qw/date_format validation_regex/];
+has carrier_description => sub { croak 'carrier_description is required' };
```

Comment:
This shouldn't be a required attribute. In fact it probably shouldn't be overridable should it? `sub carrier_description { undef }` which gets overloaded like `sub carrier_description { 'FedEx' }`. Would anyone ever want/need to change this value at run-time?